server/src/com/cloud/resource/ResourceManagerImpl.java

```diff
@@ -2378,6 +2378,23 @@ public boolean maintenanceFailed(final long hostId) {
     }
 
     @Override
+    public List<HostVO> listAllUpHosts(Type type, Long clusterId, Long podId, long dcId) {
```

Comment:
@DaanHoogland This method involves a DB query execution. I don't think this can be achieved in a unit test.